ext/standard/head.c

```diff
@@ -363,6 +363,17 @@ PHP_FUNCTION(http_response_code)
 
 	if (response_code)
 	{
+		if (SG(headers_sent) && !SG(request_info).no_headers) {
+			const char *output_start_filename = php_output_get_start_filename();
+			int output_start_lineno = php_output_get_start_lineno();
+
+			if (output_start_filename) {
+				php_error_docref(NULL, E_WARNING, "Cannot set response code - headers already sent (output started at %s:%"PRIi32")", output_start_filename, output_start_lineno);
```

Comment:
Why don't you just use `%d`? We use it everywhere else so it doesn't make any sense to use PRIi32 only here. Also please wrap the line.

Reply:
I was suggested to do so with the format string in GH-10736, so I was consistent with that.

Reply:
That's different because you log `uint32_t` there so it probably makes sense but here it is `int` so you should just use `%d`